gatsby-config.js

```diff
@@ -68,8 +68,134 @@ module.exports = {
         answer: `We have developed 4 tracks (impact areas) to assist you with building your idea. However, you're not 
         required to submit your hack to a track to win a prize!`
       }
-    ]
+    ],
+    sponsors_L: [
+      {
+        link: "https://cloud.google.com/",
+        target: "_blank",
+        rel: "noopener noreferrer",
```

Comment:
```suggestion
```
Pretty much the same in every sponsor object. Just get rid of them here and hardcode it in the template.

Reply:
Yup great suggestions I'll make the change.

Reply:
updated the PR with this update